src/plugins/intel_gpu/src/kernel_selector/kernels/strided_slice/strided_slice_kernel_ref.cpp

```diff
@@ -201,6 +201,19 @@ KernelsData StridedSliceKernelRef::GetKernelsData(const Params& params, const op
     auto dispatchData = SetDefault(newParams);
     auto entry_point = GetEntryPoint(kernelName, newParams.layerID, params, options);
     auto cldnn_jit = GetJitConstants(newParams);
+    auto input = newParams.inputs[0];
+    auto input_dt = input.GetDType();
+
+    if (!newParams.fused_ops.empty()) {
```

Comment:
Could you please clarify which primitives can be fused with strided_slice? As I can see, only Activation primitive is supported? I'll suggest to add implementation for `GetSupportedFusedOps()` function and check if primitive supported or not in `validate()` function